laterpay/__init__.py

```diff
@@ -358,6 +359,13 @@ def get_access_params(self, article_ids, lptoken=None):
             'article_id': article_ids,
         }
 
+        if muid:
+            # TODO: The behavior when lptoken and muid are given is not yet
```

Comment:
I'd recommend that we let webshell allow sending both together and then handle its own logic to determine which to use, not the python client. It will be much more maintainable than duplicating the logic in both places (and potentially having conflicting logic), and means that developers using the client don't have to update just because we decide to change webshell's priorities.